flake.nix

```diff
@@ -430,8 +430,8 @@
           # project compiles (i.e. all of `src`, `examples`, and `test`)
           packages = {
             ctl-example-bundle-web = project.bundlePursProject {
-              sources = [ "src" "examples" ];
-              main = "Examples.Pkh2Pkh";
+              sources = [ "src" "examples" "test" ];
```

Comment:
It's not important if you keep these since `bundlePursProject` accepts any number of arbitrary arguments, but since #673, this argument is no longer necessary

Reply:
You'd have to merge develop in before removing it though

Reply:
This branch is not based on develop. We will address the recent bundlePursProject-related changes in https://github.com/Plutonomicon/cardano-transaction-lib/pull/470